solution.rb

```diff
@@ -0,0 +1,19 @@
+require 'nokogiri'
+
+xml_file = File.open("macbeth.xml")
+xml_doc = Nokogiri::XML(xml_file)
```

Comment:
As part of the challenge you need to open the file from the internet, not local.